#pragma once

#include <QtWidgets/QMainWindow>
#include "ui_PtExploitDetectorGUI.h"

#define GREENCOLOR 2600544
#define REDCOLOR   15158332
#define BLUECOLOR  ‭3426654‬

class PtExploitDetectorGUI : public QMainWindow
{
    Q_OBJECT

public:
    PtExploitDetectorGUI(QWidget *parent = Q_NULLPTR);

private:
    Ui::PtExploitDetectorGUIClass ui;
    bool bigSize;
    QString cmdlineInput;
    QString messageToShow;
    void openExeToTrace();
    void startTrace();
    void dumpTrace();
    void resizeWindow();
};
